packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -375,8 +369,8 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         tabs: [for (var screen in widget.tabs) screen.buildTab(context)],
       );
       preferredSize = isNarrow
-          ? const Size.fromHeight(kToolbarHeight + 40.0)
-          : const Size.fromHeight(kToolbarHeight);
+          ? const Size.fromHeight(kToolbarHeight / 2.0 + 40.0)
```

Comment:
instead of dividing by 2 everywhere, can we add a const to the top of this file that is kToolbarHeight / 2 and use that new const everywhere we currently have `kToolbarHeight / 2.0`

Reply:
Done.